pkg/addons/assets/efa-device-plugin.yaml

```diff
@@ -0,0 +1,86 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
```

Comment:
probably worth a discussion in a separate issue, but is there a way we can ensure we always have the most up to date manifest? I might open a new issue for this as bumping images in the asset manifests is a thing that comes up reasonably often

Reply:
yes totally agree, we do this for some addons already: https://github.com/weaveworks/eksctl/blob/main/pkg/addons/default/aws_node_generate.go

Reply:
for the record, the issue is that right now, we don't have a URL to get the manifests from